packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -770,23 +770,22 @@ VirtualListBase.displayName = 'VirtualListBase';
 const VirtualListBaseNative = VirtualListBaseFactory(Native);
 VirtualListBaseNative.displayName = 'VirtualListBaseNative';
 
-const ScrollableVirtualList = ({role, ...rest}) => ( // eslint-disable-line react/jsx-no-bind
+const ScrollableVirtualList = ({...rest}) => ( // eslint-disable-line react/jsx-no-bind
 	<Scrollable
 		{...rest}
```

Comment:
Then, `{...rest} should be `{...props}`.

Reply:
Or, more simply, `(props)`.

Reply:
I missed that point. I changed from 'rest' to 'props'.